src/dotnet/APIView/APIView/Languages/CodeFileBuilder.cs

```diff
@@ -39,7 +39,6 @@ public class CodeFileBuilder
                              SymbolDisplayGenericsOptions.IncludeTypeParameters |
                              SymbolDisplayGenericsOptions.IncludeVariance,
             memberOptions: SymbolDisplayMemberOptions.IncludeExplicitInterface |
-                           SymbolDisplayMemberOptions.IncludeAccessibility |
```

Comment:
Why remove?

Reply:
This setting generates `private` modifier for the explicit implementations. Changed it to https://github.com/Azure/azure-sdk-tools/pull/1484/files#diff-f42ae8d35fb4ea2512c9d1147ec34aaa04c61724fe223eb5dfa2a4b5ef3f4ffcR507-R511